internal/cli/enable.go

```diff
@@ -0,0 +1,174 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"github.com/tilt-dev/tilt/internal/analytics"
+	engineanalytics "github.com/tilt-dev/tilt/internal/engine/analytics"
+	"github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
+	"github.com/tilt-dev/tilt/pkg/model"
+)
+
+type enableCmd struct {
+	all  bool
+	only bool
+}
+
+func newEnableCmd() *enableCmd {
+	return &enableCmd{}
+}
+
+func (c *enableCmd) name() model.TiltSubcommand { return "enable" }
+
+func (c *enableCmd) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "enable {-all | [--only] <resource>...}",
+		DisableFlagsInUseLine: true,
+		Short:                 "Enables resources",
+		Long: `Enables the specified resources in Tilt.
+
+# enables the resources named 'frontend' and 'backend'
+tilt enable frontend backend
+
+# enables frontend and backend and disables all others
+tilt enable --only frontend backend
+
+# enables all resources
+tilt enable --all
+`,
+	}
+
+	addConnectServerFlags(cmd)
+	cmd.Flags().BoolVar(&c.only, "only", false, "Enable the specified resources, disable all others")
+	cmd.Flags().BoolVar(&c.all, "all", false, "Enable all resources")
+
+	return cmd
+}
+
+func (c *enableCmd) run(ctx context.Context, args []string) error {
+	ctrlclient, err := newClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	if c.all {
+		if c.only {
+			return errors.New("cannot use --all with --only")
+		} else if len(args) > 0 {
+			return errors.New("cannot use --all with resource names")
+		}
+	} else if len(args) == 0 {
```

Comment:
we can drop the `else`
```suggestion
	} 
	if len(args) == 0 {
```